src/NServiceBus.RavenDB/Internal/DocumentStoreInitializer.cs

```diff
@@ -80,8 +116,9 @@ void EnsureClusterConfiguration(IDocumentStore store)
             }
         }
 
-        Func<ReadOnlySettings, IDocumentStore> storeCreator;
+        List<AbstractIndexCreationTask> indexesToCreate = new List<AbstractIndexCreationTask>();
```

Comment:
why `Lazy`?

Reply:
which lazy do you mean?

Reply:
Nevermind :palmface:. I've miss-read this somehow.